src/pytest/pytestTestRunner.ts

```diff
@@ -174,6 +132,27 @@ pytest.main(sys.argv[1:], plugins=[PythonTestExplorerDiscoveryOutputPlugin()])`;
         return states;
     }
 
+    private async loadEnvironmentVariables(config: IWorkspaceConfiguration): Promise<IEnvironmentVariables> {
+        const environment = await EnvironmentVariablesLoader.load(config.envFile(), process.env, this.logger);
+
+        const updatedPythonPath = [
+            config.getCwd(),
```

Comment:
Regarding the [sole difference](https://docs.pytest.org/en/stable/pythonpath.html#invoking-pytest-versus-python-m-pytest) between invoking `pytest` versus `python -m pytest`, this is explicitly handled above by also adding the current working directory to the Python path.

Reply:
> Regarding the changes around always using pytest command instead of python -m pytest. Judging by https://docs.pytest.org/en/stable/pythonpath.html#invoking-pytest-versus-python-m-pytest, in most cases, it should be fine. However, do you know if there can be any issues with that? For example, I see that it is possible that python.pythonPath can conflict with python.testing.pytestPath (imagine using global pytest and local Python from virtual env).

I don't think this would be a real concern given a Python developer who has a need for the `python.testing.pytestPath` setting. It provides a very useful integration point for test automation scenarios and external integrations, and a Python developer with that or any other use-case for it would understand its relation to the Python path.

Also, the official vscode Python plugin has supported this setting for a while, and it has not at all been a problem or point of confusion for users, but rather an important feature.

Reply:
Yeah, I think it would be better overall.